src/server/scripts/Spells/spell_mage.cpp

```diff
@@ -315,6 +320,66 @@ class spell_mage_blazing_barrier : public AuraScript
     }
 };
 
+// 190356 - Blizzard
+// 4658 - AreaTrigger Create Properties
+struct areatrigger_mage_blizzard : AreaTriggerAI
+{
+    areatrigger_mage_blizzard(AreaTrigger* areatrigger) : AreaTriggerAI(areatrigger) {}
+
+    void RefreshPeriod()
+    {
+        if (!at->GetCaster())
+            return;
+
+        _period = Milliseconds(1000);
+    }
+
+    void OnCreate() override
+    {
+        RefreshPeriod();
+    }
+
+    void OnUpdate(uint32 diff) override
+    {
+        _refreshTimer -= Milliseconds(diff);
+
+        while (_refreshTimer <= 0s)
+        {
+            if (Unit* caster = at->GetCaster())
+                caster->CastSpell(at->GetPosition(), SPELL_MAGE_BLIZZARD_DAMAGE);
+
+            RefreshPeriod();
+
+            _refreshTimer += _period;
```

Comment:
you can just do `Milliseconds(1000)` here or even better: assign the period to a constant value and then add the constant here, that way you can drop `RefreshPeriod` and `_period` since the period does not change

Reply:
Done!